app/models/gsa18f/procurement.rb

```diff
@@ -64,6 +64,10 @@ def total_price
       (cost_per_unit * quantity) || 0.0
     end
 
+    def amount
```

Comment:
I might suggest also adding the `amount` method to the `ProposalDecorator` and then calling `@proposal.amount` in the view. you can define `amount` in the decorator like this:

```ruby
def amount
  client_data.try(:amount) || ""
end
```

then you will also need to define `amount` on `WorkOrder` -- the benefit of this is that when a proposal doesn't have any client data (which won't happen in real life but does happen in tests) you won't deal with a `nil` error